namesilo.php

```diff
@@ -995,6 +995,7 @@ public function editModuleRow($module_row, array &$vars)
                     ];
                 }
             }
+			#die(var_export( $meta, true ) );
```

Comment:
It seems that you have left debugging code here.

Reply:
Oops! That's fixed now.

Reply:
There is a merge conflict that must be resolved.